### DISCUSSION TITLE
How to Disable Inertia/Overscroll Effect in Flet Column Widget?

### DISCUSSION THREAD

**Discussion by JorgeAlvarez21** [flet-dev/flet#2106] [Q&A]
### Question

I'm facing an issue with the Column widget in Flet. When scrolling to the top, it overshoots and then bounces back to the topmost item. I want to disable this inertia or overscroll effect so it stops exactly at the top without bouncing.

I haven't found a way to do this.

Thanks for your help!

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
can you make a capture of the behaviour and possibly code to reproduce..?
